programs/address-lookup-table/src/processor.rs

```diff
@@ -18,25 +18,31 @@ use {
     std::convert::TryFrom,
 };
 
-declare_process_instruction!(process_instruction, 750, |invoke_context| {
-    let transaction_context = &invoke_context.transaction_context;
-    let instruction_context = transaction_context.get_current_instruction_context()?;
-    let instruction_data = instruction_context.get_instruction_data();
-    match limited_deserialize(instruction_data)? {
-        ProgramInstruction::CreateLookupTable {
-            recent_slot,
-            bump_seed,
-        } => Processor::create_lookup_table(invoke_context, recent_slot, bump_seed),
-        ProgramInstruction::FreezeLookupTable => Processor::freeze_lookup_table(invoke_context),
-        ProgramInstruction::ExtendLookupTable { new_addresses } => {
-            Processor::extend_lookup_table(invoke_context, new_addresses)
-        }
-        ProgramInstruction::DeactivateLookupTable => {
-            Processor::deactivate_lookup_table(invoke_context)
-        }
-        ProgramInstruction::CloseLookupTable => Processor::close_lookup_table(invoke_context),
+pub const DEFAULT_COMPUTE_UNITS: u64 = 750;
+
+declare_process_instruction!(
+    process_instruction,
+    DEFAULT_COMPUTE_UNITS,
```

Comment:
the actual changes to this, and other programs, is add and use DEFAULT_COMPUTE_UNITS instead of a number (eg `750`). But the `rustfmt` rearranged large chunk of code.